src/linear_algebra/quadratic_form.lean

```diff
@@ -247,6 +375,17 @@ lemma smul_pos_def_of_nonzero {K : Type u} [linear_ordered_field K] [module K M]
   {Q : quadratic_form K M} (h : pos_def Q) {a : K} : a ≠ 0 → pos_def (a • Q) :=
 λ ha x hx, h (a • x) (λ hax, (smul_eq_zero.mp hax).elim ha hx)
 
+variables {n : Type*}
+
+lemma add_pos_def (Q Q' : quadratic_form R₂ M) (hQ : pos_def Q) (hQ' : pos_def Q') :
```

Comment:
You could change this to `pos_def.add`, so that people can use it as `hQ.add hQ'`.